includes/rest/class-server.php

```diff
@@ -0,0 +1,103 @@
+<?php
+namespace Activitypub\Rest;
+
+use WP_REST_Response;
+use Activitypub\Signature;
+
+/**
+ * ActivityPub Server REST-Class
+ *
+ * @author Django Doucet
+ *
+ * @see https://www.w3.org/TR/activitypub/#security-verification
+ */
+class Server {
+
+	/**
+	 * Initialize the class, registering WordPress hooks
+	 */
+	public static function init() {
+		\add_filter( 'rest_request_before_callbacks', array( self::class, 'authorize_activitypub_requests' ), 10, 3 );
+		\add_action( 'rest_api_init', array( self::class, 'register_routes' ) );
+	}
+
+	/**
+	 * Register routes
+	 */
+	public static function register_routes() {
+		\register_rest_route(
+			'activitypub/1.0',
```

Comment:
this is the URL that will be shown on the other platforms if we implement the catch all account, so I would create a user friendly URL and not an API endpoint?

Should we mix this up with the signature use-case?

Reply:
This is strictly for the Application acct, and is only used for Signature verification. I think I might need to give it a webfinger...

Reply:
oh ok!